.github/workflows/main.yml

```diff
@@ -64,3 +63,15 @@ jobs:
         with:
           name: drc.rpt
           path: ${{ github.workspace }}/test/drc.rpt
+
+      - name: Generate rendered images
+        uses: ./
+        with:
+          kicad_pcb: test/test.kicad_pcb
+          pcb_image: true
```

Comment:
Do we need an extra step here? Maybe just set `pcb_image: true` on the existing step at the top. Same applies to the upload step, just add the extra images path there.